logilica_weekly_report/cli_data_sources.py

```diff
@@ -43,13 +53,13 @@ def data_sources(
 
     exit_status = 0
     configuration = context.obj["configuration"]
-    logilica_credentials = context.obj["logilica_credentials"]
+    logilica_credentials = {"domain": domain}
 
     try:
-        with PlaywrightSession() as page:
+        with PlaywrightSession(headless=False) as page:
             login_page = LoginPage(page=page, credentials=logilica_credentials)
             login_page.navigate()
-            login_page.login()
+            login_page.login_with_sso()
```

Comment:
This should be somehow blocked waiting for user to actually login. Currently, this lands in debug stdout:

```
[DEBUG] lwr: Login to Logilica complete
[DEBUG] lwr: Syncing integration 'Developer Productivity' (connector 'Jira'), 0 public repositories, 0 membership-based repositories, 17 membership-based boards.
[DEBUG] lwr: Navigating to 'Settings / Integrations'
```

All this happens is printed prior the user actually login in and it is not clear from the output something is needed to be performed.

Reply:
Fixed (I _think_... 😇).

Reply:
Thanks!